client/tracks/index.js

```diff
@@ -108,12 +108,16 @@ const events = {
 		'wcpay_subscriptions_account_not_connected_product_modal_dismiss',
 	TRANSACTIONS_DOWNLOAD_CSV_CLICK: 'wcpay_transactions_download_csv_click',
 	WOOPAY_EMAIL_CHECK: 'checkout_email_address_woopay_check',
-	WOOPAY_OFFERED: 'woopay_offered',
+	WOOPAY_OFFERED: 'checkout_woopay_save_my_info_offered',
 	WOOPAY_AUTO_REDIRECT: 'checkout_woopay_auto_redirect',
 	WOOPAY_SKIPPED: 'woopay_skipped',
 	WOOPAY_BUTTON_LOAD: 'woopay_button_load',
 	WOOPAY_BUTTON_CLICK: 'woopay_button_click',
+	WOOPAY_SAVE_MY_INFO_COUNTRY_CLICK:
+		'checkout_woopay_save_my_info_country_click',
 	WOOPAY_SAVE_MY_INFO_CLICK: 'checkout_save_my_info_click',
+	WOOPAY_SAVE_MY_INFO_MOBILE_ENTER:
```

Comment:
Do we track it somewhere?

Reply:
Oof thanks for catching that. Added in https://github.com/Automattic/woocommerce-payments/pull/7973/commits/dce43950dd98d2c42c331dfd074e098001dbeb7c